packages/adaptive_scaffold/example/lib/main.dart

```diff
@@ -379,7 +518,23 @@ class _ItemList extends StatelessWidget {
       backgroundColor: const Color.fromARGB(0, 0, 0, 0),
       floatingActionButton: Breakpoints.mediumAndUp.isActive(context)
           ? null
-          : const _ComposeIcon(),
+          : Container(
```

Comment:
I had used ComposeIcon here to improve readability in relation to the actual structure of the adaptive widgets. If you've made changes to the icon you can make those changes directly to ComposeIcon. Either way it would be better to use ComposeIcon here directly.